src/components/ModelizerModelView.vue

```diff
@@ -25,21 +27,61 @@ import ComponentDefinitionsDrawer from 'src/components/drawer/ComponentDefinitio
 import { getPlugins } from 'src/composables/PluginManager';
 import PluginEvent from 'src/composables/events/PluginEvent';
 
-let pluginSubscription;
+let pluginInitSubscription;
+let pluginDeleteSubscription;
 const data = reactive({
   plugins: [],
 });
 
-onMounted(() => {
-  data.plugins = getPlugins();
-  pluginSubscription = PluginEvent.subscribe((event) => {
-    if (event === 'ready') {
-      data.plugins = getPlugins();
+/**
+ * Delete a component from a tree of components
+ *
+ * @param {Object} component - component to find and remove.
+ * @return {Boolean} true if component was found and delete otherwise false.
+ */
+function deleteComponent(component, tree) {
```

Comment:
put this in `composable/PluginManager.js`

Reply:
done